### PR TITLE
Issue #702

### LINKED ISSUE #702: Clarify array syntax and "!important"
# Motivation
Array syntax description is misleading. In particular, the examples with `!important` are not working as described:

```js
const styles = {
  button: {
    color: ['red', '!important'],
    margin: [[5, 10], '!important']
  }
}
```

Should be (note the second brackets around `'red'`):

```js
const styles = {
  button: {
    color: [['red'], '!important'],
    margin: [[5, 10], '!important']
  }
}
```

# Tasks
- [ ] Update link to array notation
- [ ] Update example

### PR DESCRIPTION
**Pull request by davidbonnet**
Fixes #702

### CODE REVIEW

File: docs/json-api.md [hunk 1]
```diff
@@ -352,7 +352,7 @@ Compiles to:
 ```javascript
 const styles = {
   button: {
-    color: ['red', '!important'],
+    color: [['red'], '!important'],
     margin: [[5, 10], '!important']
```
Comment (kof) [inline docs/json-api.md]: this one works

File: docs/json-api.md [hunk 2]
```diff
@@ -352,7 +352,7 @@ Compiles to:
 ```javascript
 const styles = {
   button: {
-    color: ['red', '!important'],
+    color: [['red'], '!important'],
```
Comment (kof) [inline docs/json-api.md]: oh I just realized, it depends on jss-expand, margin: [5, 10, '!important'] would work, but not color: ['red', '!important']

**Comment by kof** [comment]
merged, thanks

File: docs/json-api.md (hunk 2 above)
Comment (davidbonnet) [reply docs/json-api.md]: Oh, so shouldn't we document this then?

Comment (kof) [reply docs/json-api.md]: it is basically jss-expand documentation, maybe we should cut it a little bit in the json-api.md as well?

**Pull request merged**